tutorials/http_client_class.rst

```diff
@@ -29,7 +29,7 @@ It will connect and fetch a website.
 
 
         while( http.get_status()==HTTPClient.STATUS_CONNECTING or http.get_status()==HTTPClient.STATUS_RESOLVING):
-                    # Wait until resolved and connected
+        # Wait until resolved and connected
```

Comment:
Though actually, now that I read the comment, I think it should go before the while block as it explains the condition, wdyt?